python/plugins/processing/ProcessingPlugin.py

```diff
@@ -198,19 +198,21 @@ def initGui(self):
 
         self.modelerAction = QAction(
             QgsApplication.getThemeIcon("/processingModel.svg"),
-            self.tr('Graphical &Modeler...'), self.iface.mainWindow())
```

Comment:
Aren't these ellipsis correct? They open a new dialog, so should have them

Reply:
oops...

Reply:
note that if changed this should be `QCoreApplication.translate` and not `tr`…